packages/shared/styles/components/SfGallery.scss

```
@@ -116,3 +116,14 @@ $gallery__stage-cursor: zoom-in !default;
     user-select: none;
   }
 }
+
+.glide__zoom {
```

all component classes should have respective prefix:

```suggestion
.sf-gallery__zoom {
```

---

Okey, I am going to change all the classes so that they have a prefix according to the component to which they belong, in this case to SfGallery